src/Lykke.Service.IroncladDecorator/OpenIdConnect/IOpenIdValidators.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading.Tasks;
+using IdentityModel.Client;
+using Lykke.Service.IroncladDecorator.Sessions;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+
+namespace Lykke.Service.IroncladDecorator.OpenIdConnect
+{
+    public interface IOpenIdValidators
```

Comment:
This interface makes not sense for me. I would suggest to split into two validators.

Reply:
Fixed